test/cloudtest/pkg/commands/main.go

```diff
@@ -1030,7 +1051,7 @@ func (ctx *executionContext) generateClusterFailedReportEntry(inst *clusterInsta
 
 func (ctx *executionContext) generateTestCaseReport(test *testTask, totalTests int, totalTime time.Duration, failures int, suite *reporting.Suite) (int, time.Duration, int) {
 	testCase := &reporting.TestCase{
-		Name: test.test.Name,
+		Name: test.test.Key,
```

Comment:
I do not think it is required here.

Reply:
It is not required but it gives more information in Test Case Report. Right now it shows just first cluster of the list

Reply:
ok